File: JavaScript/1005.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var firstInput = parseFloat(lines.shift());
var secondInput = parseFloat(lines.shift());

var media = ((firstInput*3.5) + (secondInput*7.5))/11;
media = media.toFixed(5);

console.log("MEDIA = " + media);

File: Python/1005.py
peso1 = 2
peso2 = 3
peso3 = 5
a = float(input())
b = float(input())
c = float(input())

result = "MEDIA = " + '%.1f' % (((a*peso1)+(b*peso2)+(c*peso3))/(peso1+peso2+peso3))
print(result)
